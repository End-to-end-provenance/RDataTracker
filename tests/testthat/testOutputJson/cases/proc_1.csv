"ddg.type","ddg.num","ddg.name","ddg.value","ddg.return.linked","ddg.time","ddg.snum","ddg.startLine","ddg.startCol","ddg.endLine","ddg.endCol"
"Start",1,"ReturnTest.R","ReturnTest.R",FALSE,1.42,NA,NA,NA,NA,NA
