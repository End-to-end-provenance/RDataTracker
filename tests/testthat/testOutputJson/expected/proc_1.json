
	"activity" : {
		// procedure nodes
		"rdt:p1": {
			"rdt:name": "ReturnTest.R",
			"rdt:type": "Start",
			"rdt:elapsedTime": 1.42,
			"rdt:scriptNum": "NA",
			"rdt:startLine": "NA",
			"rdt:startCol": "NA",
			"rdt:endLine": "NA",
			"rdt:endCol": "NA"
		}
	}
